elasticsearch/utils/es_load_kibana_ui_objects

```diff
@@ -50,6 +50,11 @@ cat $INDEX_PATTERN_FILE | \
 info Adding the Kibana UI objects . . .
 {
     for file in $KIBANA_UI_OBJECTS_DIR/*.json ; do
+        if [ ! -f "$file" ] ; then
```

Comment:
So this is the case where the glob does not match anything so `file` gains the value of `$KIBANA_UI_OBJECTS_DIR/*.json`, right?